onset_maproom/agronomy.py

```diff
@@ -2,10 +2,9 @@
 
 def soil_plant_water_balance(
     sm_yesterday,
-    rain,
+    peffective,
     et,
-    taw,
-    runoff=0
+    taw
```

Comment:
A little python tidbit: a comma is allowed after the last item in a list, and in multi-line lists I usually put one, because then when you add or remove the last element the resulting diff shows one line changed instead of two.